tests/utils/LHNTestUtils.js

```diff
@@ -167,6 +170,63 @@ function getAdvancedFakeReport(isArchived, isUserCreatedPolicyRoom, hasAddWorksp
     };
 }
 
+/**
+ * @param {Number[]} [participantAccountIDs]
+ * @param {Number} [millisecondsInThePast] the number of milliseconds in the past for the last message timestamp (to order reports by most recent messages)
+ * @param {boolean} [isUnread]
+ * @returns {Object}
+ */
+function getFakeReportWithPolicy(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false) {
+    return {
+        ...getFakeReport(participantAccountIDs, millisecondsInThePast, isUnread),
+        type: CONST.REPORT.TYPE.CHAT,
+        chatType: CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT,
+        policyID: '08CE60F05A5D86E1',
+        oldPolicyName: '',
+        isOwnPolicyExpenseChat: false,
+        ownerAccountID: 1,
```

Comment:
```suggestion
        ownerAccountID: participantAccountIDs[0],
```
should this depend on `participantAccountIDs` instead?

Reply:
@techievivek you're right, I changed it in this commit [751a380c](https://github.com/Expensify/App/pull/30068/commits/751a380c3e85ee12c299e1ec5122d2553785031c)